spec/services/hyrax/collection_member_service_spec.rb

```diff
@@ -1,28 +1,19 @@
 RSpec.describe Hyrax::CollectionMemberService, :clean_repo do
-  let(:work_attrs) { { id: '123', title_tesim: ['A generic work'] } }
-
-  let(:coll1_attrs) { { id: 'col1', title_tesim: ['A Collection 1'], child_object_ids_ssim: [work.id] } }
-  let(:coll2_attrs) { { id: 'col2', title_tesim: ['A Collection 2'], child_object_ids_ssim: [work.id, 'abc123'] } }
-  let(:coll3_attrs) { { id: 'col3', title_tesim: ['A Collection 3'], child_object_ids_ssim: ['abc123'] } }
-
+  let(:user) { create(:user, groups: 'potato') }
+  let!(:ability) { ::Ability.new(user) }
+  let!(:col1) { build(:collection_lw, id: 'col1', user: user, with_solr_document: true) }
+  let!(:col2) { build(:collection_lw, id: 'col2', with_solr_document: true) }
```

Comment:
Excellent use of new light weight factory.

Reply:
:) So convenient too!